### ISSUE TITLE
no-case-headers CLI flag has odd behaviour

### ISSUE THREAD

**Issue by mowies** [citycide/tablemark-cli#7]
While making use of the `--no-case-headers` CLI flag, I got this output:
```
> tablemark some-input-file.json --no-case-headers
The option "case-headers" is unknown. Did you mean the following one?
-N, --no-case-headers  Disable automatic sentence casing of derived key names (disabled by default)
```

The `-N` shorthand works as expected but the long version doesn't work for me.

**Comment by haltcase** [comment]
Thanks for the issue @mowies. That certainly isn't working as intended. That flag's kind of weird since it's there to disable an option that's enabled by default in the `tablemark` library, which is also why the description is so confusing ( `Disable ... (disabled by default)` ) now that I read it again.

`args` has some magic to automatically negate boolean flags with a `--no-` prefix, so it's probably interfering with our parameter. I've been working on a new release since releasing `tablemark` v3 that replaces `args` and makes this flag work properly. I'll work on polishing that up for release - stay tuned!

**Comment by haltcase** [comment]
Apologies for the delay, but this is now released in [v3.0.0](https://github.com/haltcase/tablemark-cli/releases/tag/v3.0.0).
